src/Event.php

```diff
@@ -132,33 +133,33 @@
     private $stacktrace;
 
     /**
-     * Class constructor.
-     *
-     * @param Configuration $config The client configuration
+     * Event constructor.
      */
-    public function __construct(Configuration $config)
+    public function __construct()
     {
-        $this->id = Uuid::uuid4();
+        try {
+            $this->id = Uuid::uuid4();
+        } catch (\Exception $e) {
+            // TODO: This happens when Uuid::uuid4 has issues, fallback to something else?
```

Comment:
The problem is that if we do that, we add `@throws` everywhere imho it's really annoying to have a `@throws` in the ctor of `Event`.

Reply:
I understand that it may be annoying to have IDEs warn that an exception could be thrown by the constructor but it's foundamentally broken to catch a failed attempt to create an UUID as an event without it is not valid. Imho btw the ID should be generated server-side (what happens if someone generates an already existing UUID?) and then be returned by the request so that it can be set back to the event object in the client. Also if in the future you want to change the format of the field every SDK will have to handle this. I understand that anyway this is out of scope for this PR

Reply:
I removed the `try/catch`